### PR TITLE
Teachers and teaching

### PR DESCRIPTION
**Pull request by braunms**
- elaborate mindset of teachers 
- add online learning platforms as source for curriculum and teachers

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -812,38 +812,55 @@ In order to set up a proper educational scheme we need to discuss two more items
...
-
-- "If you want to go fast, go alone; if you want to go far, go together?"
-- Not every "good" scientist wants to become a "good" software engineer, too!
-- The Carpentries philosophy [@Duckles2016]
+Irrespective of where people come from they need to have the proper mindset to properly foster RSEs-to-be.
+The basis for our proposed mindset is the conviction that research software engineering (RSEing) is an important topic that supports good scientific practice and reproducible research as well as the motivation to share their experience and skills with prospective RSEs. 
+In addition, we expect it is a fundamental part of a teacher's mindset to adopt a set of shared values, which we have detailed above in the [values](#values) section.
+Good teachers engage with the different scientific backgrounds of the learners, regardless of their prior knowledge.
+In order to identify relevant content and adapt content to the learners needs, they understand and appreciate the respective research environment.
+We would presume good teachers to take responsibility in improving their own educational skills to make their teaching more effective.
```
Comment (mhagdorn) [inline paper_teaching-learning-RSE.md:836]: "pedagogical" instead of "educational"

File: paper_teaching-learning-RSE.md [hunk 2]
```diff
@@ -812,38 +812,55 @@ In order to set up a proper educational scheme we need to discuss two more items
...
+For example, a survey by The Carpentries [@Duckles2016] provides a detailed evaluation on the value and benefits of Software and Data Carpentry workshops to their instructors.
+The expected mindset also encompasses that teachers should see setbacks and feedback as an opportunity to learn and grow their own skills.
+A teacher's mindset should also include the acknowledgment that some learners acquire additional RSE skills out of necessity and not out of interest, meaning that not every "good" scientist wants to become a "good" software engineer, too.
+Therefore, it is also inevitable that teachers have positive and high expectations of their students.
+Inspired by the proverb "If you want to go fast, go alone; if you want to go far, go together?", teachers should actively engage with the RSE community and see the benefits of networking.
+Finally, teachers should be motivated to follow up and offer continued support to their students after the training has ended.
 
 #### Where do we get our teachers from
-The community discussion brought about the need for a mixture of people, thereby 
+The community discussion brought to light the need for a mixture of people, thereby
```
Comment (mhagdorn) [inline paper_teaching-learning-RSE.md:846]: "therefore" rather than "thereby"

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (braunms) [reply paper_teaching-learning-RSE.md]: Thanks! Resolved in commit195862c

File: paper_teaching-learning-RSE.md (hunk 2 above)
Comment (braunms) [reply paper_teaching-learning-RSE.md]: Thanks! Resolved in commit 195862c

**Pull request merged**